packages/apps/composer-app/src/stories/NavtreePresence.stories.tsx

```diff
@@ -99,7 +99,7 @@ export const Demo = {
                 // @ts-ignore
                 presence: ({ data: { object } }: { data: { object: any } }) => {
                   return (
-                    <ObjectPresence size={2} viewers={object?.viewers ?? []} classNames={[fineBlockSize, 'is-6']} />
+                    <SmallPresence size={2} members={object?.viewers ?? []} classNames={[fineBlockSize, 'is-6']} />
```

Comment:
Generally I’d prefer a concept like `small` to be applied as an enum value of a prop like `variant` or `size` and be wrapped by a component that defers to internal components as needed, but this is fine for now.

Reply:
Yeah, tbh I think these might even be two separate components or the small one might not even be long for this world so I left this close-ish to what it was previously. Makes sense though 👍